File: fib.py
init = parser.add_argument('--n', type=int)
args = parser.parse_args()
init =  args.n
___
def isBase(curr):
	return curr == 0 or curr == 1
___
def fBase(curr):
	return curr
___
lambda n: n-1 
lambda n: n-2
___
def merge(subproblems): return sum(subproblems)


# Format of this file
# Base values
# ___
# Subproblems - n is the current number
# ___
# Function to merge the subproblems together

File: fib.ml
let init = 500000;;
___
let isBase curr =
	curr == 0 || curr == 1
;;
___
let fBase curr =
	curr
;;
___
(fun n -> n - 1)
(fun n -> n - 2)
___
let rec sum l =
	match l with
	| h::t ->
		h + (sum t)
	| [] ->
		0
;;
let merge subproblems = 
	sum subproblems
;;
___
%d